### PR TITLE
Config implementation

### PR DESCRIPTION
**Pull request by Tyler1800**
It implements config and does some stuff with shuffleboard

NSP

### CODE REVIEW

File: src/main/deploy/devConfig.txt [hunk 1]
```diff
@@ -0,0 +1,123 @@
+shooterSpeed = -0.37
```
Comment (Gray5878) [inline src/main/deploy/devConfig.txt]: do we really want all the old stuff here

File: src/main/java/frc/robot/Configrun.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
+package frc.robot;
```
Comment (Gray5878) [inline src/main/java/frc/robot/Configrun.java:1]: can we get rid of the old comments?

File: src/main/java/frc/robot/Subsystems/Swerve/Drivetrain.java [hunk 3]
```diff
@@ -16,7 +16,7 @@
 import edu.wpi.first.wpilibj.controller.PIDController;
 import edu.wpi.first.wpilibj2.command.*;
 import edu.wpi.first.wpilibj.Timer;
-
+import frc.robot.Configrun;
```
Comment (Gray5878) [inline src/main/java/frc/robot/Subsystems/Swerve/Drivetrain.java]: Do we need to load Configrun here?

**Comment by Tyler1800** [comment]
Hey, can you re review this plz

File: src/main/deploy/protoConfig.txt [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+backLeftTurningEncoderPort = 0
+backRightTurningEncoderPort = 1
+
+# Motor Offsets in Radians
+frontLeftOffset = 0.0
+frontRightOffset = 0.7853
+backLeftOffset = -0.0884
+backRightOffset = 2.79
+
+# RIP rickroll
```
Comment (Gray5878) [inline src/main/deploy/protoConfig.txt:25]: RIP

File: src/main/java/frc/robot/Configrun.java (hunk 2 above)
Comment (Gray5878) [reply src/main/java/frc/robot/Configrun.java:1]: thanks

**Review by Gray5878 — APPROVED** [review]
5 out of 5
good code, RIP rickroll

**Pull request merged**